### PR TITLE
Add possibility to change user

### PR DESCRIPTION
**Pull request by fpcurie**
Due to problem with nfsv4 with a local user.

### CODE REVIEW

File: src/run.sh [hunk 1]
```diff
@@ -24,6 +24,18 @@ getEnv() {
     redis_host=${REDIS_HOST:-redis}
     redis_port=${REDIS_PORT:-6379}
     ipv6=${ENABLE_IPV6:-false}
+    elabftw_user=${ELABFTW_USER:-nginx}
+    elabftw_group=${ELABFTW_GROUP:-nginx}
+    elabftw_userid=${ELABFTW_USERID:-100}
+    elabftw_groupid=${ELABFTW_GROUPID:-101}
+}
+
+# Create user if not exit
```
Comment (NicolasCARPi) [inline src/run.sh:33]: please fix typo here

File: src/run.sh [hunk 2]
```diff
@@ -24,6 +24,18 @@ getEnv() {
...
+createUser() {
+    if [ "${elabftw_user}" != "nginx" ]; then
+        addgroup -g ${elabftw_groupid} ${elabftw_group}
+        adduser -S -u ${elabftw_userid} -G s-elabftw-gp ${elabftw_user}
```
Comment (NicolasCARPi) [inline src/run.sh:37]: `s-elabftw-gp` should be `${elabftw_group}` here.

**Comment by NicolasCARPi** [comment]
Please also sign the CLA: https://cla-assistant.io/elabftw/elabimg

File: src/run.sh (hunk 1 above)
Comment (fpcurie) [reply src/run.sh]: Fixed

File: src/run.sh (hunk 2 above)
Comment (fpcurie) [reply src/run.sh]: Oups, fixed too :)

**Comment by fpcurie** [comment]
> 
> 
> Please also sign the CLA: https://cla-assistant.io/elabftw/elabimg

Done ;)

**Comment by NicolasCARPi** [comment]
Thanks LordFP ;)

**Pull request merged**